src/api/public/apidocs-new/paths/source_project_name_cmd_addchannels.yaml

```
@@ -0,0 +1,16 @@
+post:
+  summary: Add channels to a source
```

```suggestion
  summary: Add channels to a source.
```
Let's finish sentences with a dot.